Examples/DeveloperCocoapodsExample/AppcuesCocoapodsExample.entitlements

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
from what I observe, this gets set to production regardless in a TestFlight build, but I don't think it really matters here

Reply:
Yeah it does. So I think `development` is ok here for tests in the simulator